### ISSUE TITLE
Edit the README file

### ISSUE THREAD

**Issue by mahdifani14** [mahdifani14/AutoTuning#4]
Please complete the README file with the dependency installation procedures and important hints what you have been facing with, thanks.

**Comment by pooyaww** [comment]
Yes, Sir! But,let's focus on main issues, read me file can be presented at last stage.
